### PR TITLE
masksd 386d to 390d

### PR DESCRIPTION
**Pull request by TheWonderfulTartiflette**
- [ ] 386d
- [ ] 387d
- [ ] 388d
- [ ] 389d
- [ ] 390d

### CODE REVIEW

File: masksd/00386.png
[the remark this answers is not in the document]
Comment (TheWonderfulTartiflette) [reply masksd/00386.png:1]: Ah thanks, I'll fix that, and definitely not rush them like I did next time. I was wondering, how do you use the same template between different images ?

**Comment by AlexandreSato** [comment]
I have two methods:

1 - I choose an imgsd with good contrast in the windows and make a masksd like "MyCar template" and save it as 5 ready-made masksd then open a draftPR to be able to see it in the reviewer (and then make the definitive mask)

2 - I save a single image of myCarTemplate on github and use the "rawcontent" link on imglabeller in the import pre-existing mask field

Em 7 de jun. de 2023, à(s) 17:14, The Wonderful Tartiflette ***@***.***> escreveu:

﻿

@TheWonderfulTartiflette commented on this pull request.

________________________________

On masksd/00386.png<https://github.com/commaai/comma10k/pull/3334#discussion_r1222107900>:

Ah thanks, I'll fix that, and definitely not rush them like I did next time. I was wondering, how do you use the same template between different images ?

File: masksd/00386.png
[the remark this answers is not in the document]
Comment (TheWonderfulTartiflette) [reply masksd/00386.png:1]: > Welcome back glad we had help! Sorry had more things to fix and ended up passing beaten last time
> 
> On masksd 386: In this rear window I believe it is a car parked by the wall image
> 
> In this little corner I believe it is still the arm image
> 
> The previous masks all consider this area to be "undrivable" image
> 
> On masksd 387: The same in this area, my strategy has been to use the same "myCar template" mask on all of them before I start labeling image

Ah thanks, I'll fix that, and definitely not rush them like I did next time. I was wondering, how do you use the same template between different images ?

**Comment by AlexandreSato** [comment]
Step by step:
1 - I go to spreadsheet and search the last one:
image


2 - After i go to imgsd folder and search for 352 to see the respective PR:
image
in this case is PR=3292


3 - I go to reviewer and put the "3292" here:
image


4 - In my case i liked this imgsd because have a good contrast (black / white) in the windows:
image


5 - So i made one "my car template":
image


 6 - An copy:
image


7 - Rename:
image


8 - Create a separated branch clicking here:
image


9 - Upload the files:
image


10 - And i create a PR for you to continue and fell how easy is not start from scratch:
https://github.com/commaai/comma10k/pull/3336

**Pull request closed without merging**